user/contactmethod/queries.sql

```diff
@@ -0,0 +1,132 @@
+-- name: AddContactMethod :exec
```

Comment:
Can we prefix these (e.g., ContactMethodAdd)? It will make it easier to maintain/use since all queries end up in a single package.